drivers/google/google.go

```diff
@@ -396,7 +396,11 @@ func (d *Driver) Remove() error {
 	}
 
 	if err := c.deleteInstance(); err != nil {
-		return err
+		if strings.Contains(err.Error(), "notFound") {
```

Comment:
We should be able to cast this to the `googleapi.Error` type and check the `Code` field for a `http.StatusNotFound` code instead of relying on string matching.

Reply:
Ok, i think i need some help with that. These were my first few lines in go.

I tried to change it, but I'm not sure at all, that it will work, is correct, or is clean code.

Reply:
You aced it as far as I can tell :)

Thank you!